### PR TITLE
Fixed typos and string inconsistencies in Empyrean Campaign.

### PR DESCRIPTION
**Pull request by gunchleoc**
Fixed typos and string inconsistencies in Empyrean Campaign.

Note: This branch should only be merged after the release, because it will break the string freeze/translations.

### CODE REVIEW

File: mods/empyrean_campaign/cutscenes/ending.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ music=music/unrest_theme.ogg
 image=images/menus/backgrounds/badlands.png,1
 caption=At long last, your journey across the surface and into the Underworld has come to an end.
 pause=8s
-caption=Upon entering The Pit, a glowing obelisk shined in the distance.
+caption=Upon entering The Pit, a glowing obelisk shone in the distance.
```
Comment (dorkster) [inline mods/empyrean_campaign/cutscenes/ending.txt]: This is not an error. Both "shined" and "shone" are valid past-tense forms of shine.

**Comment by dorkster** [comment]
Since these are just English typos, they can probably be fixed pre-1.0.
I'll take care of this if we decide to do it.

**Comment by gunchleoc** [comment]
The problem is that Gettext now won't fetch the translations for these strings - the English strings are the search keywords for translations, which now won't match any more. So, if you merge this now, we will have to create new .pot files and ask everybody to translate again before the release.

File: mods/empyrean_campaign/cutscenes/ending.txt (hunk 1 above)
Comment (gunchleoc) [reply mods/empyrean_campaign/cutscenes/ending.txt]: There is a difference in meaning though. "shone" is what the sun did, "shined" is what the shoe-shine boy did. The obelisk is emitting a shine itself, not making something else shine.

At least that's how I am familiar with the word - maybe this distinction gut blurred during recent years.

**Comment by gunchleoc** [comment]
Never mind, I just looked at the diff and you have taken care of it :)

**Pull request closed without merging**